sentinel/src/transitions/update_notifications.js

```diff
@@ -105,78 +94,47 @@ module.exports = {
     return new Promise((resolve, reject) => {
       var self = module.exports,
         doc = change.doc,
-        patient_id = doc.fields && doc.fields.patient_id,
         config = module.exports.getConfig(),
-        eventType = getEventType(config, doc);
+        eventType = getEventType(config, doc),
+        patient;
 
       if (!eventType) {
         return resolve();
       }
 
+      logger.info('`update_notifications` transitions is deprecated. Please use `muting` transition instead');
```

Comment:
I think this would be better in the `init()` function so it warns on startup rather than every time the transition runs.